src/framework/uicomponents/qml/MuseScore/UiComponents/StyledDialogView.qml

```diff
@@ -63,7 +63,7 @@ DialogView {
     }
 
     onOpened: {
-        navSec.requestActive()
+        Qt.callLater(navSec.requestActive)
```

Comment:
@cbjeukendrup 
Do we really need to call later the section activation?
Could you please explain what problem did you solve?

Because of this decision, it is now necessary to delay the activation of the control in all dialogs.
see https://github.com/musescore/MuseScore/pull/10906/commits/ade54f2e86b979e0a537b495e7aefc4d00c14668

Reply:
It was necessary in order to fix the activation when opening the "Save to cloud" dialog from the "How would you like to save?" dialog. I tried it now and the fix is still necessary. 

Now that I look at it again, I see that the reason that it wasn't working, was that there was a `callLater` in `onClosed` (a few lines below), to activate the parent control. If we do that without `callLater`, then we also don't need `callLater` in `onOpened`.